lib/beaker/options/parser.rb

```diff
@@ -205,7 +205,8 @@ def update_option(key, value, source)
       #   3.  the 'CONFIG' section of the hosts file
       #   4.  options file values
       #   5.  subcommand options, if executing beaker subcommands
-      #   6.  default or preset values are given the lowest priority
+      #   6.  options from $HOME/.beaker/subcommands_options.yaml
```

Comment:
@rishijavia does this step (reading `$HOME/.beaker/...`) happen on all beaker executions (both a full beaker run & in subcommands)? We definitely don't want `subcommand_options.yaml` (looks like both subcommand & options are plural, that's not right, right @tvpartytonight ?) to be read in the context of a full beaker run.